doc/option_params/decimal_numeric.rb

```diff
@@ -0,0 +1,7 @@
+require 'optparse'
+include OptionParser::Acceptables
+parser = OptionParser.new
+parser.on('--decimal_numeric=DECIMAL_numeric', DecimalNumeric) do |value|
```

Comment:
The name of the value in the option descriptor is mixed-case, is this by design?

Reply:
No, an editing error.  Will fix in next commit.